mod_reforged/config/perk_defs.nut

```diff
@@ -924,6 +924,14 @@
 		Icon = "ui/perks/rf_alert.png",
 		IconDisabled = "ui/perks/rf_alert_bw.png",
 	},
+	{
+		ID = "perk.rf_supporter",
+		Script = "scripts/skills/perks/perk_rf_supporter",
+		Name = ::Const.Strings.PerkName.RF_Supporter,
+		Tooltip = ::Const.Strings.PerkDescription.RF_Supporter,
+		Icon = "ui/perks/rf_supporter.png",
+		IconDisabled = "ui/perks/rf_supporter_sw.png",
```

Comment:
Remove comma

Reply:
Every other perk def introduced by reforged also ends with a comma here

Reply:
Yeah that's a mistake that we are going to fix in a separate overall commit later, but for the new perkdefs we shouldn't add the comma.